Gopkg.toml

```diff
@@ -53,3 +53,7 @@
 [[constraint]]
   name = "github.com/docker/go"
   version = "1.5.1-1"
+
+[[constraint]]
```

Comment:
Looking at https://github.com/pivotal/image-relocation -- would it make sense to have an initial release we can pinpoint here?

Not necessarily blocking this PR though.

Reply:
Possibly, although that would make most sense if go-containerregistry was also released, which it hasn't been so far. Let's defer for now.

Reply:
@glyn  - For helm3 the push/pull utilizes https://github.com/deislabs/oras which is based on containerd @bacongobbler do you think there could be a release of oras for CNAB if needed?

Reply:
Thanks. The plan is to merge this PR with its implementation in terms of go-containerregistry. The implementation can be changed in the future as appropriate.